pkg/tagger/collectors/docker_extract.go

```diff
@@ -88,6 +91,23 @@ func dockerExtractLabels(tags *utils.TagList, containerLabels map[string]string,
 		case "io.rancher.stack_service.name":
 			tags.AddLow("rancher_service", labelValue)
 
+		// Custom labels as tags
+		case "com.datadoghq.ad.tags":
+			tagNames := []string{}
+			err := json.Unmarshal([]byte(labelValue), &tagNames)
+			if err != nil {
+				log.Debugf("Cannot unmarshal AD tags: %s", err)
+			}
+			for _, tag := range tagNames {
+				tagParts := strings.Split(tag, ":")
+				// skip if tag is not in expected k:v format
+				if len(tagParts) != 2 {
+					log.Debugf("Tag %s is not in k:v format", strings.Join(tagParts, ", "))
```

Comment:
maybe the debug log can just display the input `tag` string
```suggestion
					log.Debugf("Tag '%s' is not in k:v format",tag)
```